impeller/entity/save_layer_utils.cc

```diff
@@ -0,0 +1,86 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "impeller/entity/save_layer_utils.h"
+
+namespace impeller {
+
+std::optional<Rect> ComputeSaveLayerCoverage(
+    const Rect& content_coverage,
+    const Matrix& effect_transform,
+    const Rect& coverage_limit,
+    const std::shared_ptr<FilterContents>& image_filter,
+    bool flood_output_coverage,
+    bool flood_input_coverage) {
+  Rect coverage = content_coverage;
+  // There are three conditions that currently trigger flood_input_coverage, the
```

Comment:
But currently only one condition sets this flag, which is !!BDF

Reply:
Noted this here